test/utils/isSameDay_spec.js

```diff
@@ -15,6 +15,15 @@ describe('isSameDay', () => {
     expect(isSameDay(today, tomorrow)).to.equal(false);
   });
 
+  it('returns false for same days of week', () => {
+    // Flags accidentally use of moment's day() function, which returns index
```

Comment:
This spec is to guard against a [bug I recently introduced and reverted](https://github.com/airbnb/react-dates/pull/414#discussion-diff-109070922L6). Moment is inconsistent about day vs date, and I think our naming makes sense.